core/templates/dev/head/pages/topic_editor/questions/QuestionsTabDirective.js

```diff
@@ -23,22 +23,31 @@ oppia.directive('questionsTab', [
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/pages/topic_editor/questions/questions_tab_directive.html'),
       controller: [
-        '$scope', '$uibModal', 'TopicEditorStateService', 'TopicUpdateService',
-        'UndoRedoService', 'UrlInterpolationService',
-        'EVENT_STORY_SUMMARIES_INITIALIZED', 'EVENT_TOPIC_INITIALIZED',
-        'EVENT_TOPIC_REINITIALIZED',
+        '$scope', 'AlertsService', 'TopicEditorStateService',
+        'QuestionCreationService', 'EditableQuestionBackendApiService',
+        'QuestionObjectFactory', 'EVENT_QUESTION_SUMMARIES_INITIALIZED',
         function(
-            $scope, $uibModal, TopicEditorStateService, TopicUpdateService,
-            UndoRedoService, UrlInterpolationService,
-            EVENT_STORY_SUMMARIES_INITIALIZED, EVENT_TOPIC_INITIALIZED,
-            EVENT_TOPIC_REINITIALIZED) {
-          // TODO(aks681): Continue with the questions tab for topic editor here
-          // and create another questions editor directive to be used here.
+            $scope, AlertsService, TopicEditorStateService,
+            QuestionCreationService, EditableQuestionBackendApiService,
+            QuestionObjectFactory, EVENT_QUESTION_SUMMARIES_INITIALIZED) {
           var _initTab = function() {
+            $scope.showEditor = false;
```

Comment:
$scope.editorIsShown = false; (don't start with verb)

Also, what editor is this? May want to make it more explicit -- I can't tell whether it's questionEditorIsShown or questionEditorTabIsShown or something else.

Reply:
Done, renamed to questionEditorIsShown.